ddtrace/telemetry/__init__.py

```diff
@@ -0,0 +1,47 @@
+import six
+
+import ddtrace.internal.telemetry.telemetry_writer
+
+
+class _InstrumentationTelemetryStatus(type):
```

Comment:
Why does this class exist? It seems like the method should just be on `InstrumentationTelemetry` 🤔 

I do see that runtime metrics did this but not sure why this pattern was used.

Reply:
I removed this meta class. It seems unnecessary